.circleci/config.yml

```diff
@@ -59,9 +65,9 @@ jobs:
           name: Build extension - Firefox Cross-browser UI
           command: yarn build:cross-browser-ui:firefox
 
-      - run:
-          name: Build extension - Chrome Cross-browser UI
-          command: yarn build:cross-browser-ui:chrome
+      #- run:
```

Comment:
I'll add a comment in the code or move this change to outside this PR: Chrome version is broken since the engine is hardcoded to use the wormhole which doesn't exist on CHrome.